app/services/waste_exemptions_engine/regular_charging_strategy.rb

```diff
@@ -0,0 +1,24 @@
+# frozen_string_literal: true
+
+module WasteExemptionsEngine
+  class RegularChargingStrategy < ChargingStrategy
+
+    private
+
+    def initial_compliance_charge(band)
+      band == highest_band ? band.initial_compliance_charge.charge_amount : 0
+    end
+
+    def additional_compliance_charge(band, initial_compliance_charge_applied)
+      # SonarCloud complains about unused parameters but we need them for inheritance
+      # But if we use them to placate SonarCloud, Rubocop complains about void contexts
+      # rubocop:disable Lint/Void
+      initial_compliance_charge_applied
```

Comment:
not sure what `initial_compliance_charge_applied` doing here. Is that a boolean or integer? As far as I see it's not being used in any calculations in this method, so what is it for?

Reply:
See the comment above. If I don't reference it, SonarCloud complains about an unused parameter. If I do reference it, rubocop complains about void contexts. So I reference it and disable the void cop.